test/integration/rules/label/label.html

```diff
@@ -5,67 +5,85 @@
   <input type="submit" id="na4" />
   <input type="reset" id="na5" />
   <input type="text" id="fail1" />
-  <textarea id="fail3"></textarea>
+  <textarea id="fail2"></textarea>
   <input type="text" aria-label="label" id="pass1" />
-  <textarea aria-label="label" id="pass3"></textarea>
-  <input type="text" aria-labelledby="label" id="pass4" />
-  <textarea aria-labelledby="label" id="pass6"></textarea>
+  <textarea aria-label="label" id="pass2"></textarea>
+  <input type="text" aria-labelledby="label" id="pass3" />
+  <textarea aria-labelledby="label" id="pass4"></textarea>
   <div id="label">Label</div>
-  <label>Label <input type="text" id="pass7" /></label>
-  <label>Label <textarea id="pass9"></textarea></label>
+  <label>Label <input type="text" id="pass5" /></label>
+  <label>Label <textarea id="pass6"></textarea></label>
 
-  <label><input type="text" id="fail4" /></label>
-  <label><textarea id="fail6"></textarea></label>
-  <label for="fail7"></label><input type="text" id="fail7" />
-  <label for="fail9"></label><textarea id="fail9"></textarea>
+  <label><input type="text" id="fail3" /></label>
+  <label><textarea id="fail4"></textarea></label>
+  <label for="fail5"></label><input type="text" id="fail5" />
+  <label for="fail6"></label><textarea id="fail6"></textarea>
 
-  <label for="fail11" style="display: none">Text</label
-  ><input type="text" id="fail11" /> <label for="pass10">Label</label
-  ><input type="text" id="pass10" /> <label for="pass12">Label</label
-  ><textarea id="pass12"></textarea>
+  <label for="fail7" style="display: none">Text</label
+  ><input type="text" id="fail7" /> <label for="pass7">Label</label
+  ><input type="text" id="pass7" /><label for="pass8">Label</label
+  ><textarea id="pass8"></textarea>
 
-  <input id="pass13" title="Label" />
-  <textarea id="pass15" title="Label"></textarea>
+  <input id="pass9" title="Label" />
+  <textarea id="pass10" title="Label"></textarea>
 
   <label
-    ><label><input type="text" id="fail22" /></label
+    ><label><input type="text" id="fail8" /></label
   ></label>
 
   <div>
-    <label>
```

Comment:
This label does nothing, and fail24 is not tested by this rule; this is not the select-name rule.